### PR TITLE
Revert "Set min version to 1.10 (#1932)"

### PR DESCRIPTION
**Pull request by elezar**
This reverts commit b28736a2870449cbfcdf72415d73df4fcb20c2a2.

### CODE REVIEW

File: frameworks/helloworld/tests/test_placement.py [hunk 1]
```diff
@@ -49,14 +49,14 @@ def fault_domain_vars_are_present(pod_instance):
...
     options = {
-        'service': {
-            'spec_file': 'examples/marathon_constraint.yml'
+        "service": {
+            "spec_file": "examples/marathon_constraint.yml"
         },
-        'hello': {
-            'placement': '[["rack_id", "LIKE", "rack-foo-.*"]]'
+        "hello": {
+            "placement": "[[\\\"rack_id\\\", \\\"LIKE\\\", \\\"rack-foo-.*\\\"]]"
```
Comment (gabrielhartmann) [inline frameworks/helloworld/tests/test_placement.py]: Why are these placement changes included in this PR?

Comment (benclarkwood) [reply frameworks/helloworld/tests/test_placement.py]: Trying to weed my way through the Cosmos differences in 1.9 and 1.10+

Going to try this locally and see what's up.

**Comment by orsenthil** [comment]
ping! - this PR is blocking 1.9 of `dcos/dcos`

**Comment by benclarkwood** [comment]
@orsenthil it will be over the line shortly.

**Comment by benclarkwood** [comment]
@orsenthil 1.9 should be unblocked.

**Pull request merged**